build-tc

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Installing yarn
+YARN_VERSION="0.23.2"
+TOOLS_DIRECTORY="tools"
+
+if [ ! -d "$TOOLS_DIRECTORY" ]; then
```

Comment:
Note that if the version is ever bumped then this won't upgrade yarn to the specified version. This may be surprising to future travellers.